packages/angular_devkit/core/src/analytics/index.ts

```diff
@@ -25,7 +25,7 @@ export enum NgCliAnalyticsDimensions {
   RamInGigabytes = 3,
   NodeVersion = 4,
   NgAddCollection = 6,
-  NgBuildBuildEventLog = 7,
+  NgBuildBuildEventLog = 7, // deprecated
```

Comment:
I marked this enum as deprecated instead of deleting it because deleting would mess up the dimension IDs in https://github.com/angular/angular-cli/blob/master/docs/design/analytics.md